mail-manager/src/test/scala/it/pagopa/interop/commons/mail/MailerImplSpec.scala

```diff
@@ -1,63 +1,62 @@
 package it.pagopa.interop.commons.mail
 
-import it.pagopa.interop.commons.mail.model.{MailAttachment, MailData, MailDataTemplate}
-import it.pagopa.interop.commons.mail.service.InteropMailer
-import it.pagopa.interop.commons.mail.service.impl.DefaultInteropMailer
-import it.pagopa.interop.commons.utils.model.TextTemplate
 import org.jvnet.mock_javamail.Mailbox
 import org.scalatest.concurrent.PatienceConfiguration.Timeout
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.time.{Seconds, Span}
 import org.scalatest.wordspec.AnyWordSpecLike
-import it.pagopa.interop.commons.MockMailerConfiguration
-
-import java.util.UUID
+import Mail._
+import javax.mail.internet.InternetAddress
+import javax.mail.Message
 import javax.mail.internet.MimeMultipart
 
 class MailerImplSpec extends AnyWordSpecLike with Matchers with ScalaFutures {
 
-  val mailer: InteropMailer = new DefaultInteropMailer with MockMailerConfiguration
+  val mailer: InteropMailer = InteropMailer.unsafeFrom(mail"mock-sender@interop.pagopa.it", Mocks.mockedMailer())
 
   val timeout: Timeout = Timeout(Span(3, Seconds))
 
   "a MailSender" should {
-    "send templated text email" in {
-
-      val uuidToken = UUID.randomUUID()
-      val party     = "Comune di Milano"
-
-      val mailData = MailDataTemplate(
-        recipients = Seq("legal@comune.milano.it"),
-        subject = TextTemplate("Interop - Onboarding ${partyName}", Map("partyName" -> party)),
-        body = TextTemplate("This is the UUID token ${token}", Map("token" -> uuidToken.toString)),
+    "send a text email" in {
+      val mail: Mail = Mail(
+        recipients = Seq(mail"legal@comune.milano.it"),
```

Comment:
to avoid weird and unwanted behaviours, may we use totally fake addresses in tests? 
(I know they haven't changed, but I've never noticed)